### PR TITLE
add Darkmode

### PR DESCRIPTION
**Pull request by Tealk**
adds automatic change to darkmode depending on the system theme

### CODE REVIEW

**Comment by RikudouSage** [comment]
The navbar doesn't look that great, but it looks much better if you add `navbar-dark` class.

The best way to do it would be:

1. add a property called `darkModeEnabled` (like [here](https://github.com/Fediseer/FediseerGUI/blob/master/src/app/app.component.ts#L35), but it will be public, so something like `public darkModeEnabled: boolean = false`)
2. Add these two properties to the html navbar element:
  1. `[class.navbar-dark]="darkModeEnabled"`
  2. `[class.navbar-light]="!darkModeEnabled"`
3. Add `this.darkModeEnabled = darkModeDetected` to app.component.ts

This will automatically add the required classes based on the value of the  `darkModeEnabled` property.

Also, could you take a look at changing the colors of the links for dark mode? There might be another similar class missing because the blue links on dark background look horrible.

**Comment by Tealk** [comment]
> Also, could you take a look at changing the colors of the links for dark mode? There might be another similar class missing because the blue links on dark background look horrible.

what color would you like? I find that you can read blue well on both themes.
Here they use the same blue: https://adminlte.io/docs/3.1/

File: src/app/app.component.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export class AppComponent implements OnInit {
 
   private autoCollapse = 992;
 
+  public darkModeEnabled: boolean = false
```
Comment (RikudouSage) [inline src/app/app.component.ts:33]: Missing a semicolon:

```suggestion
  public darkModeEnabled: boolean = false;
```

**Comment by RikudouSage** [comment]
It's really straining my eyes reading the blue on the dark background, even in the link you provided.

**Comment by Tealk** [comment]
better? i try to stay with blue
image

**Comment by RikudouSage** [comment]
Yes! Much better.

**Pull request merged**